charts/session-manager/templates/api-pub/configmap.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "template.name" . }}-api-pub-config
+  labels:
+    {{- include "template.labels" . | nindent 4 }}
+    app.kubernetes.io/component: api-pub
+immutable: {{ .Values.config.isImmutable | default false }}
+data:
+  config.yaml: |-
+    application:
+      name: {{ .Chart.Name }}
+    {{- with .Values.config }}
+      environment: {{ .environment }}
+      {{- with .labels }}
+      labels:
+        {{- toYaml . | nindent 8 }}
+      {{- end}}
+
+    http:
+      {{- toYaml .http | nindent 6 }}
+
+    grpc:
+      {{- toYaml .grpc | nindent 6 }}
+
+    status:
+      {{- toYaml .status | nindent 6 }}
+
+    logger:
+      {{- toYaml .logger | nindent 6 }}
+
+    telemetry:
+      {{- toYaml .telemetry | nindent 6 }}
+
+    database:
+      {{- toYaml .database | nindent 6 }}
+
+    sessionManager:
+      {{- toYaml .sessionManager | nindent 6 }}
+    {{- end }}
```

Comment:
The cm deleted by this PR clearly distinguishes between configuration based on github.com/openkcm/common-sdk/pkg/commoncfg and github.com/openkcm/session-manager/internal/config. I'd prefer to keep this structure.